sql/hive/pom.xml

```diff
@@ -185,6 +157,10 @@
       <groupId>org.apache.derby</groupId>
       <artifactId>derby</artifactId>
     </dependency>
+    <dependency>
+      <groupId>jline</groupId>
+      <artifactId>jline</artifactId>
```

Comment:
ditto, why handle `jline` separately?

Reply:
Spark already declared `jline` dependency (with the same version as that from Hive) so I excluded it from the shaded jar and added it here as a transitive dependency of `spark-hive` module.

Reply:
Does spark use datanucleus directly? If no, I think we can put it into uber jar with relocation.